install-overlay

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -euo pipefail
+cd "$(dirname "$0")"
+
+set -x
+mkdir -p $HOME/.config/nixpkgs/overlays
+ln -sf $PWD/overlay.nix $HOME/.config/nixpkgs/overlays/nur.nix
```

Comment:
At the moment I have 

```nix
packageOverrides = {
  nur = callPackage (import (builtins.fetchGit {
    url = "https://github.com/nix-community/NUR";
  })) {};
};
```

in my `~/.config/nixpkgs/config.nix`
Do you think an overlay would be better?

Reply:
In the end we could also advertise different methods to use nur?

Reply:
Agreed, there are more possible installation methods and we should document them all. That was just the first one that came to mind.

I am not sure what the TTL for fetchGit is, does it pull on every nix evaluation?

Reply:
It updates every hour. Just like tarballs.